projects/canopy/src/lib/button/button.notes.ts

```diff
@@ -16,23 +16,50 @@ Import the component in your module:
 
 and in your HTML:
 
-~~~
+~~~html
 <button lg-button type="button" [variant]="solid-primary">Button</button>
 ~~~
 
 or:
 
-~~~
+~~~html
 <a lg-button href="#" [variant]="primary">Link</a>
 ~~~
 
+## Button with icon with text
+
+Icons can be content projected into the button, the button component will then handle the styling of that icon.
+The \`ButtonIconPosition\` property can be used to locate the icon on the left hand side of the text, the default is to the right hand side.
+Content projection is used as Canopy Icons need loading via the iconRegistry which enables tree shaking of icons.
+
+~~~html
+<button lg-button type="button" [ButtonIconPosition]="left" >
+  Add item
+  <lg-icon name="add" />
+</button>
+~~~
+
+## Button with icon only
+
+Buttons can be setup as icon only buttons, this is done in a similar way to a button with icon and text.
+You will still need to include text content inside the button as this information is vital to screen readers.
+The \`iconButton\` property is used to visually hide that text from sighted users.
+
+~~~html
+<button lg-button type="button" [iconButton]="true" >
+  Add item
+  <lg-icon name="add" />
+</button>
+~~~
+
 ## Inputs
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
 | \`\`variant\`\` | The variant of button: \`\`solid-primary\`\`, \`\`solid-secondary\`\`, \`\`outline-primary\`\`, \`\`outline-secondary\`\`, \`\`reverse-primary\`\`, \`\`reverse-secondary\`\`; | string | primary | No |
 | \`\`fullWidth\`\` | If the button has to span full width or not. For 'sm' and 'md' sized screens, the button will always be full width and this input has no affect | boolean | false | No |
 | \`\`loading\`\` | If the button shows a loading spinner and is also disabled | boolean | false | No |
+| \`\`iconPosition\`\` | The position of the icon in the button | string | right | No |
```

Comment:
I think `iconButton` is missing from this table

Reply:
Added 👍